aries-site/src/data/wcag/skeleton.json

```diff
@@ -0,0 +1,513 @@
+    },
+    {
+      "rule": "2.4.10",
+      "label": "Section Headings",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.4.11",
+      "label": "Focus Not Obscured (Minimum)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.4.12",
+      "label": "Focus Not Obscured (Enhanced)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.4.13",
+      "label": "Focus Appearance",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.1",
+      "label": "Pointer Gestures",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.2",
+      "label": "Pointer Cancellation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.3",
+      "label": "Label in Name",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.4",
+      "label": "Motion Actuation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.5",
+      "label": "Target Size (Enhanced)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.6",
+      "label": "Concurrent Input Mechanisms",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.7",
+      "label": "Dragging Movements",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "2.5.8",
+      "label": "Target Size (Minimum)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.1",
+      "label": "Language of Page",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.2",
+      "label": "Language of Parts",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.3",
+      "label": "Unusual Words",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.4",
+      "label": "Abbreviations",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.5",
+      "label": "Reading Level",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.1.6",
+      "label": "Pronunciation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.2.1",
+      "label": "On Focus",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.2.2",
+      "label": "On Input",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.2.3",
+      "label": "Consistent Navigation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "3.2.4",
+      "label": "Consistent Identification",
+      "status": "conditional",
+      "notes": ""
+    },
+    {
+      "rule": "3.2.5",
+      "label": "Change on Request",
+      "status": "not-applicable",
```

Comment:
conditional 
When the skeleton is replaced by the actual content, ensure this transition is expected by the user and doesn't introduce sudden changes to the UI that could cause confusion

Reply:
Do you feel there is something we can be doing for this or is this really in the domain of the application where they are used? We really don't have enough information to be all that helpful (ie. we can't say what we're waiting for to be loaded etc.)

Reply:
I think the way I was thinking about this one was a skeleton component should only appear when a user clicks a button, or navigates to a section or something that is user requested. So if a skeleton was shown like on a auto -refresh I would think it may not pass since the skeleton would be unexpected

Reply:
I feel like this may fall slightly outside of the scope of the Skeleton component and be more of an application design thing but I would be okay with marking as conditional to help flag for people using a skeleton component that this is something they should be aware of.

Reply:
Agree that it'd probably be important to flag regardless.

Reply:
@KennyAtHPE do you wanna just add a note in the notes here